src/registrar/templates/emails/domain_request_withdrawn.txt

```diff
@@ -0,0 +1,69 @@
+{% autoescape off %}{# In a text file, we don't want to have HTML entities escaped #}
+Hi {{ application.submitter.first_name }}.
+
+Your .gov domain request has been withdrawn.
+DOMAIN REQUESTED: {{ application.requested_domain.name }}
+REQUEST #: {{ application.id }}
+STATUS: Withdrawn
+
+
+YOU CAN EDIT YOUR WITHDRAWN REQUEST
+The details of your withdrawn request are included below. You can edit and resubmit this application by logging into the registrar. <https://manage.get.gov/>.
+THANK YOU
+.Gov helps the public identify official, trusted information. Thank you for requesting a .gov domain.
+
+----------------------------------------------------------------
+SUMMARY OF YOUR DOMAIN REQUEST
```

Comment:
I know this PR is a WIP so sorry for jumping the gun: There's a template for SUMMARY that you can plug in here instead of laying out all the fields.

Reply:
@rachidatecs omg this is a good call, are you talking about `{% include 'emails/includes/application_summary.txt' %}
` 😭